src/main/java/baritone/Baritone.java

```diff
@@ -122,6 +123,7 @@ public synchronized void init() {
             exploreProcess = new ExploreProcess(this);
             backfillProcess = new BackfillProcess(this);
             farmProcess = new FarmProcess(this);
+            sortProcess = new ChestSortProcess(this); this.getGameEventHandler().registerEventListener(sortProcess);
```

Comment:
due to the formatting here i have wreaked havok on my undergarments

Reply:
rekt